client/js/shout.js

```diff
@@ -190,13 +190,39 @@ $(function() {
 		var chan = chat.find(target);
 		var from = data.msg.from;
 
+		var msg = $(render("msg", {messages: [data.msg]}));
 		chan.find(".messages")
-			.append(render("msg", {messages: [data.msg]}))
+			.append(msg)
 			.trigger("msg", [
 				target,
 				data.msg
 			]);
 
+		var text = msg.find('.text');
+		if (text.find('i').size() === 1) {
+			text = text.find('i');
+		}
+		// Channels names are strings (beginning with a '&' or '#' character)
+		// of length up to 200 characters.
+		// See https://tools.ietf.org/html/rfc1459#section-1.3
+		text.html(text.html().replace(/(^|\s)([#&][^\x07\x2C\s]{0,199})/ig,
+				'$1<span class="inline-channel" role="button" tabindex="0" data-chan="$2">$2</span>'));
```

Comment:
@AmShaegar13 good point. #547 got merged , you don't have to fix this one, rebasing will drop the warning for this one.